### DISCUSSION TITLE
About the right click "send to txt2img" function

### DISCUSSION THREAD

**Discussion by rayadaschn** [zanllp/infinite-image-browsing#384] [Q&A] [answered]
Hello, this project is great, but there is some confusion about the project.
How is the right click "send to txt2img" function to synchronize to SD-webui implemented?
I see that there is a websockt polling. So the underlying mechanism is websockt synchronization?
Also, if I open more than one SD-webui, I find that the "send to txt2img" is synchronized across multiple pages.
I'm curious to see how this piece of functionality is implemented. Any help you can provide would be greatly appreciated.

**Comment by zanllp** [comment, marked as answer]
BroadcastChannel + Click the hidden button

https://github.com/zanllp/sd-webui-infinite-image-browsing/blob/345709bcde169a428fb8592d9fea0b91f309ba46/javascript/index.js#L97-L138

https://github.com/zanllp/sd-webui-infinite-image-browsing/blob/345709bcde169a428fb8592d9fea0b91f309ba46/scripts/iib_setup.py#L19-L64
